### ISSUE TITLE
Installation issue while setting the api for any exchange, this is the error I received.

### ISSUE THREAD

**Issue by tymiles003** [bakman2/cryptofolio#14]
Fatal error: Uncaught Error: Call to undefined function ccxt\curl_init() in /var/www/html/folio/lib/php/Exchange.php:495 Stack trace: #0 /var/www/html/folio/portfolio.php(24): ccxt\Exchange->__construct(Array) #1 /var/www/html/folio/index.php(9): require_once('/var/www/html/f...') #2 {main} thrown in /var/www/html/folio/lib/php/Exchange.php on line 495

**Comment by bakman2** [comment]
Has the exchange/api/secret been stored in config.json ?

Did you follow this step ?
> - Make sure config.json is writable with the correct owner:group and make sure the directory and file can be accessed by your webserver

**Comment by tymiles003** [comment]
Yes, and moved out of the web directory and setup in the config.php file as well.

**Comment by bakman2** [comment]
It seems that you are missing the curl extension for PHP.

**Comment by tymiles003** [comment]
php-curl is already the newest version (1:7.2+59+ubuntu16.04.1+deb.sury.org+1).

**Comment by bakman2** [comment]
`Call to undefined function ccxt\curl_init()`
Is not compatible with your version ?

**Comment by tymiles003** [comment]
What version should I use?

**Comment by bakman2** [comment]
I am using ccxt for the APIs, requirements according to ccxt (not version specific):

>It requires common PHP modules:

>cURL
>mbstring (using UTF-8 is highly recommended)
>PCRE
>iconv

**Comment by tymiles003** [comment]
I'll reinstall it on a new server and report my findings.

**Comment by tymiles003** [comment]
Question, 
How do I change the default currency to USD?

**Comment by bakman2** [comment]
At the moment you can't. I working on a refactor of the whole project, also including multiple currencies.

**Comment by tymiles003** [comment]
ok, looking forward to it.
Thanks

**Comment by tymiles003** [comment]
Fresh install of ubuntu 16.04, with  curl (7.47.0-1ubuntu2.5) and  php7.0-curl (7.0.22-0ubuntu0.16.04.1) and I continue to get the error below??


Fatal error: Uncaught Error: Call to undefined function ccxt\mb_split() in /var/www/html/cryptofolio/lib/php/Exchange.php:618 Stack trace: #0 /var/www/html/cryptofolio/lib/php/Exchange.php(607): ccxt\Exchange->define_rest_api(Array, 'request') #1 /var/www/html/cryptofolio/portfolio.php(24): ccxt\Exchange->__construct(Array) #2 /var/www/html/cryptofolio/index.php(9): require_once('/var/www/html/c...') #3 {main} thrown in /var/www/html/cryptofolio/lib/php/Exchange.php on line 618

**Comment by tymiles003** [comment]
I figured it out, missing some php modules.

**Comment by tymiles003** [comment]
Sir,
ETA on the usd currency option?
Thanks
